src/index.js

```diff
@@ -66,7 +66,8 @@ const filesProto = {
   statByPath: files.statByPath,
   downloadById: files.downloadById,
   downloadByPath: files.downloadByPath,
-  getDownloadLink: files.getDownloadLink,
```

Comment:
@gregorylegarec You're right, I can quickly fix in Photos but maybe for Files that can be more complicated in this sprint's end. I'll add it 👍